setup.py

```diff
@@ -142,7 +142,7 @@
         'SLIST = obspy.io.ascii.core',
         'PICKLE = obspy.core.stream',
         'CSS = obspy.io.css.core',
-        'DATAMARK = obspy.io.datamark.core',
+        'WIN = obspy.io.win.core',
```

Comment:
Hmm.. In general I agree that we can make an exception here and allow the module move/renaming without deprecation reroute, as people have no reason to use the low level routines.

But.. we could still break peoples' codes if they circumvent the filetype detection, i.e. if they do `read(..., format='DATAMARK')`.

Maybe we should leave the `DATAMARK` plugin in there (pointing to new `io.win`)?

Also, maybe we should wait with merging until it's clear how this one and #1692 will play together?